index.html

```diff
@@ -1,21 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="css/main.css">
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <div class="contener">
+        <div class="text_area_nav">
+        <div class="text_nav1">
+            <div class="scouer"></div>
+
+            <h1 class="title_nav">Maya Nelson</h1>
+        </div>
+        <div class="text_nav2">
+            <h2>PROJECT MANAGER</h2>
```

Comment:
You don't really need to wrap the only h2 in a div since it is alone just move the class from the div to the h2